File: snippets/php/handle_parsed_emails.php
function handleInboundEmailWebhook($req)
{
    // Get the webhook events from the request body. We know
    // every event will be an inbound event, because we're
    // only using this endpoint for inbound email.
    $mandrill_events = $req->body;
    $inboundEvents = $mandrill_events->inboundEvents;

    foreach ($inboundEvents as $inboundEvent) {
        $msg = $inboundEvent->msg;
        $form_email = $msg->from_email;
        $subject = $msg->subject;
        $text = $msg->text;

        $user = $fakeDb->getUserByEmail($form_email);

        $fakeDb->createTodo(
            [
                "user" => $user,
                "text" => $subject,
                "description" => $text
            ]
        );
    }
}

File: snippets/python/handle_parsed_emails.py
from operator import itemgetter

def handleInboundEmailWebhook(request):
  try:
    mandrill_events = itemgetter('mandrill_events')(request)
    for inboundEvent in mandrill_events['inboundEvents']:
      from_email = inboundEvent['from_email']
      subject = inboundEvent['subject']
      text = inboundEvent['text']
      msg = inboundEvent['msg']
      user = fakeDb.getUserByEmail(from_email)
      fakeDb.createTodo({
      'user' : user,
      'text' : subject,
      'description' : text
    })
  except: 
    print('Empty request passed')

handleInboundEmailWebhook()